share/spice/bike_sharing/indego_phl/content.handlebars

```diff
@@ -0,0 +1,15 @@
+<div class="tile__body">
```

Comment:
We can remove this wrapping div. The template already wraps all this in a `tile__body` div :+1: